### PR TITLE
Update Profiles, Itinerary, Img, FAQ's, and Removed Banner

### PR DESCRIPTION
**Pull request by davidBubela**
- Reinstated Guliette's original profile 
- Fixed Tanya's photo to remove white space
-  Faq updated to have slack hyperlink
- Itinerary schedule updated
- removed banner

### CODE REVIEW

File: src/components/RescheduleUpdateHomepage.tsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ import { Panel } from './utils/Panel'
 import { Itinerary } from './sections/Itinerary'
 import { Title } from './utils/Title'
 import { Outro } from './sections/Outro'
-import { RescheduleUpdateIntro } from './sections/RescheduleUpdateIntro'
+// import { RescheduleUpdateIntro } from './sections/RescheduleUpdateIntro'
```
Comment (exvuma) [inline src/components/RescheduleUpdateHomepage.tsx:19]: I'd just delete it, no reason to keep old code around in comments just as a hygiene thing

File: src/components/sections/Faq.tsx [hunk 2]
```diff
@@ -110,6 +110,15 @@ export const faqs = slugifyFaqs([
       </p>
     ),
   },
+  {
+    title: "How do I coordinate with folks?",
+    body: (
+      <p>
+        The best way to communicate is to join our slack <a href='https://turnonthefawcetts.slack.com/join/invite/enQtMjQ1MDMzNjY5MDkxOS05OWFhMzAxZDkwZDFlYzJiZmFmODZmNDg1ZWE4YWE0MTg1MzcyMDBlNmEwZDY4ZTgzNDAwZTA3MjFhYTgyMDYz#/credentials'>here!</a>
```
Comment (exvuma) [inline src/components/sections/Faq.tsx:117]: Shit, is there any way to put this under the protected path john set up at  https://turnonthefawcetts.com/mvps  ? 

I don't want everyone having that link

Comment (exvuma) [reply src/components/sections/Faq.tsx:117]: Sorry that was not in reqs my bad

File: src/components/sections/Itinerary.tsx [hunk 3]
```diff
@@ -7,13 +7,18 @@ import { Box } from 'reflexbox'
...
+  {}, //empty column for seperation & aesthetics
+  { description: 'Getting Ready (Wedding Party)', location: 'TBD', date: 'October 22nd, 2021', time: '10 - 2pm' },
+  { description: 'Ceremony', location: 'Lighthouse', date: 'October 22nd, 2021', time: '3pm' },
+  { description: 'Cocktail Hour', location: 'Firepits', date: 'October 22nd, 2021', time: '4pm' },
+  {}, //empty column for seperation & aesthetics
+  { description: 'Photos for Party', location: 'Dock + Other', date: 'October 22nd, 2021', time: '4pm' },
+  { description: 'Reception', location: 'Sky Terrace', date: 'October 22nd, 2021', time: '6pm' },
+  // {
+  //   description: 'Group Event',
+  //   time: 'TBD',
```
Comment (exvuma) [inline src/components/sections/Itinerary.tsx:19]: yeah, def def the comments. Git has all the history we need built in 👍

File: src/components/sections/Itinerary.tsx [hunk 4]
```diff
@@ -28,13 +33,14 @@ export const Itinerary: React.FC = () => (
           >
             Let us know
           </a>{' '}
-          which events you can make it to.
+          which events you can make it to by downloading and accepting the slack invite!
```
Comment (exvuma) [inline src/components/sections/Itinerary.tsx:36]:
```suggestion
          which events you can make it to on the sheet we sent in the email.
```

a nit

File: src/profiles.tsx [hunk 5]
```diff
@@ -300,4 +301,25 @@ export const profiles = [
       </>
     ),
   },
+  {
+    name: 'Guliette Bernard',
```
Comment (exvuma) [inline src/profiles.tsx:305]: Show me what this looks like? Share a screenshot. She is not a bridesmaid.. Here is my spiel 

> Wasn’t Victoria’s sister a bridesmaid? 
No... Victoria did not kick Guliette - her middle sister - out of the wedding. Turmoil over the last 5 years has existed between the two. Guliette threatened to cancel at a bad time late in the game. The show went on. Guliette decided to lovingly apologize in a very sweet, creative ways that were not necessary but appreciated.

**Review by exvuma — CHANGES_REQUESTED** [review]

File: src/profiles.tsx (hunk 5 above)
Comment (davidBubela) [reply src/profiles.tsx:305]: That's my fault for not catching BM role. Peep the screenshots and let me know if I should keep it. 
guliettePartyCard
gulietteSoloCard

TLDR: Removing her role left an awkward card spacing so I assigned a new role to her that fit the best IMO

Comment (davidBubela) [reply src/profiles.tsx:305]: simplySister
sisterOfThe

1 or 2

**Review by exvuma — APPROVED** [review]
